contrib/completion/bash/docker

```diff
@@ -5486,7 +5486,7 @@ _docker_wait() {
 	_docker_container_wait
 }
 
-COMPOSE_PLUGIN_PATH=$(docker info --format '{{json .ClientInfo.Plugins}}' | sed -n 's/.*"Path":"\([^"]\+docker-compose\)".*/\1/p')
+COMPOSE_PLUGIN_PATH=$(docker info --format '{{json .ClientInfo.Plugins}}' | sed -n 's/.*"Path":"\([^"]\{1,\}docker-compose\)".*/\1/p')
```

Comment:
Can you please check whether `sed -n -E 's/.*"Path":"([^"]+docker-compose)".*/\1/p'` also works?
I'd rather avoid the range modifier here in favour of the more commonly used and easiser to comprehend `+`.

Reply:
@albers Yes, it also works. Actually, it was my first solution, but I thought that I should follow the motives of the author of the original change.

Reply:
@albers I've force pushed suggested changes (sed with extended regexp enabled). Thanks for review